README.md

```diff
@@ -1,7 +1,12 @@
 ![](http://i.imgur.com/fYp148T.jpg)
+
+#### Important! ####
+On 2021-06-15 freenode removed the corsix-th channel on IRC causing migration off the network. We have added a matrix space bridged to the existing discord server. If you would like us to register on another IRC server, **please open a new issue**.
```

Comment:
I'm just wary of time-limited information, I always forget to go back and remove it when no longer needed.